src/regedit/register.jl

```diff
@@ -1,25 +1,25 @@
 """
 Return a `GitRepo` object for an up-to-date copy of `registry`.
 """
-function get_registry(registry::String; gitconfig::Dict=Dict())
+function get_registry(registry::AbstractString; gitconfig::Dict=Dict())
     reg_path(args...) = joinpath("registries", map(string, args)...)
     if haskey(REGISTRIES, registry)
         registry_uuid = REGISTRIES[registry]
         registry_path = reg_path(registry_uuid)
         if !ispath(registry_path)
-            LibGit2.clone(registry, registry_path, branch="master")
```

Comment:
Whats the reason for this change? Using `LibGit2` stuff makes things quite a bit faster.

Reply:
LibGit2 was not using my SSH key. I'm not super attached to keeping it this way but it made development easier for me.

Reply:
Okay, fair enough.